contents/blog/hiring-myths-and-truths.md

```diff
@@ -0,0 +1,75 @@
+---
+date: 2022-01-20
+title: Basic myths and truths about hiring that startups keep getting wrong
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - charles-cook
+featuredImage: ../images/blog/posthog-ceo-diary-blog.png
+featuredImageType: full
+category: Startups
+---
+
+These are some of the myths and truths I've learned about hiring for startups over the past decade. I hope they will help you turn recruitment into a more tolerable and less frustrating experience. 
```

Comment:
I like a short intro, but this a little too minimal. Maybe lean into your credentials a bit more? Why should someone trust your opinion about hiring?